.changeset/pretty-bats-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': minor
```

Comment:
I've been marking all changes to TreeView as patch changes since it's still a "draft"

```suggestion
'@primer/react': patch
```